packages/xo-web/src/xo-app/user/index.js

```diff
@@ -282,6 +289,84 @@ const SshKeys = addSubscriptions({
   )
 })
 
+// ===================================================================
+const dateFormat = d3.timeFormat('%Y-%m-%d %H:%M')
+
+const COLUMNS_AUTH_TOKENS = [
+  {
+    itemRenderer: ({ created_at }) => {
+      if (created_at !== undefined) {
+        return dateFormat(created_at)
+      }
+      return _('notDefined')
+    },
+    name: _('authTokenCreatedAt'),
+    sortCriteria: 'created_at',
+  },
+  {
+    default: true,
+    itemRenderer: ({ expiration }) => dateFormat(expiration),
+    name: _('authTokenExpiration'),
+    sortCriteria: 'expiration',
+  },
+  {
+    itemRenderer: ({ id }) => (
+      <Copiable tagName='pre' data={id}>
+        {id.slice(4, 9)}
+      </Copiable>
+    ),
+    name: _('authToken'),
+  },
+  {
+    itemRenderer: token => (
+      <Text value={token.description} onChange={description => editAuthToken({ ...token, description })} />
+    ),
+    name: _('authTokenDescription'),
+  },
+]
```

Comment:
IMHO, I believe this order of columns would be better:
1. token (most important element)
2. description (variable content size)
3. creation
4. expiration
5. actions